spec/controllers/groups_controller_spec.rb

```diff
@@ -73,7 +73,7 @@
 
   describe 'GET all_people' do
     let(:group) { create(:group, valid_attributes) }
-    let!(:people) { instance_double(Person::ActiveRecord_Relation) }
+    let!(:people) { instance_double(Person.const_get(:ActiveRecord_Relation)) }
```

Comment:
saw lots of replacements  by using 'const_get',  then I did a few searches and got to know this issues more 🤓